shippable test results are too long for comments

https://github.com/ansible/ansible/pull/21195

https://app.shippable.com/github/ansible/ansible/runs/16468/1/tests

```
5 DOCUMENTATION.options.rm.description.0: expected a list @ data['options']['rm']['description'][0]. Got 'Remove the zip file, after unzipping'\nlib/ansible/modules/windows/win_unzip.py:0:0: E305 DOCUMENTATION.options.src.description.0: expected a list @ data['options']['src']['description'][0]. Got 'File to be unzi
pped (provide absolute path)'\nlib/ansible/modules/windows/win_updates.py:0:0: E305 DOCUMENTATION.options.category_names.description.0: expected a list @ data['options']['category_names']['description'][0]. Got 'A scalar or list of categories to install updates from'\nlib/ansible/modules/windows/win_updates.py:0:0: 
E305 DOCUMENTATION.options.log_path.description.0: expected a list @ data['options']['log_path']['description'][0]. Got 'If set, win_updates will append update progress to the specified file. The directory must already exist.'\nlib/ansible/modules/windows/win_updates.py:0:0: E305 DOCUMENTATION.options.state.descript
ion.0: expected a list @ data['options']['state']['description'][0]. Got 'Controls whether found updates are returned as a list or actually installed.'\nlib/ansible/modules/windows/win_updates.py:0:0: E305 DOCUMENTATION.options.state.description.1: expected a list @ data['options']['state']['description'][1]. Got 'T
his module also supports Ansible check mode, which has the same effect as setting state=searched'\nlib/ansible/modules/windows/win_uri.py:0:0: E305 DOCUMENTATION.options.body.description.0: expected a list @ data['options']['body']['description'][0]. Got 'The body of the HTTP request/response to the web service.'\nl
ib/ansible/modules/windows/win_uri.py:0:0: E305 DOCUMENTATION.options.content_type.description.0: expected a list @ data['options']['content_type']['description'][0]. Got 'Sets the \"Content-Type\" header.'\nlib/ansible/modules/windows/win_uri.py:0:0: E305 DOCUMENTATION.options.dest.description.0: expected a list @ 
data['options']['dest']['description'][0]. Got 'Output the response body to a file.'\nlib/ansible/modules/windows/win_uri.py:0:0: E305 DOCUMENTATION.options.headers.description.0: expected a list @ data['options']['headers']['description'][0]. Got 'Key Value pairs for headers. Example \"Host: www.somesite.com\"'\nli
b/ansible/modules/windows/win_uri.py:0:0: E305 DOCUMENTATION.options.method.description.0: expected a list @ data['options']['method']['description'][0]. Got 'The HTTP Method of the request or response.'\nlib/ansible/modules/windows/win_uri.py:0:0: E305 DOCUMENTATION.options.url.description.0: expected a list @ data
['options']['url']['description'][0]. Got 'HTTP or HTTPS URL in the form of (http|https)://host.domain:port/path'\nlib/ansible/modules/windows/win_uri.py:0:0: E305 DOCUMENTATION.options.use_basic_parsing.description.0: expected a list @ data['options']['use_basic_parsing']['description'][0]. Got \"This module relies
 upon 'Invoke-WebRequest', which by default uses the Internet Explorer Engine to parse a webpage. There's an edge-case where if a user hasn't run IE before, this will fail. The only advantage to using the Internet Explorer praser is that you can traverse the DOM in a powershell script. That isn't useful for Ansible,
 so by default we toggle 'UseBasicParsing'. However, you can toggle that off here.\"\nlib/ansible/modules/windows/win_user.py:0:0: E305 DOCUMENTATION.options.account_disabled.description.0: expected a list @ data['options']['account_disabled']['description'][0]. Got 'C(yes) will disable the user account.  C(no) will
 clear the disabled flag.'\nlib/ansible/modules/windows/win_user.py:0:0: E305 DOCUMENTATION.options.account_locked.description.0: expected a list @ data['options']['account_locked']['description'][0]. Got 'C(no) will unlock the user account if locked.'\nlib/ansible/modules/windows/win_user.py:0:0: E305 DOCUMENTATION
.options.description.description.0: expected a list @ data['options']['description']['description'][0]. Got 'Description of the user'\nlib/ansible/modules/windows/win_user.py:0:0: E305 DOCUMENTATION.options.fullname.description.0: expected a list @ data['options']['fullname']['description'][0]. Got 'Full name of the
 user'\nlib/ansible/modules/windows/win_user.py:0:0: E305 DOCUMENTATION.options.groups.description.0: expected a list @ data['options']['groups']['description'][0]. Got \"Adds or removes the user from this comma-separated lis of groups, depending on the value of I(groups_action). When I(groups_action) is C(replace) 
and I(groups) is set to the empty string ('groups='), the user is removed from all groups.\"\nlib/ansible/modules/windows/win_user.py:0:0: E305 DOCUMENTATION.options.groups_action.description.0: expected a list @ data['options']['groups_action']['description'][0]. Got 'If C(replace), the user is added as a member of
 each group in I(groups) and removed from any other groups.  If C(add), the user is added to each group in I(groups) where not already a member.  If C(remove), the user is removed from each group in I(groups).'\nlib/ansible/modules/windows/win_user.py:0:0: E305 DOCUMENTATION.options.name.description.0: expected a li
st @ data['options']['name']['description'][0]. Got 'Name of the user to create, remove or modify.'\nlib/ansible/modules/windows/win_user.py:0:0: E305 DOCUMENTATION.options.password.description.0: expected a list @ data['options']['password']['description'][0]. Got \"Optionally set the user's password to this (plain
 text) value.\"\nlib/ansible/modules/windows/win_user.py:0:0: E305 DOCUMENTATION.options.password_expired.description.0: expected a list @ data['options']['password_expired']['description'][0]. Got 'C(yes) will require the user to change their password at next login. C(no) will clear the expired password flag.'\nlib
/ansible/modules/windows/win_user.py:0:0: E305 DOCUMENTATION.options.password_never_expires.description.0: expected a list @ data['options']['password_never_expires']['description'][0]. Got 'C(yes) will set the password to never expire.  C(no) will allow the password to expire.'\nlib/ansible/modules/windows/win_user
.py:0:0: E305 DOCUMENTATION.options.state.description.0: expected a list @ data['options']['state']['description'][0]. Got 'When C(present), creates or updates the user account.  When C(absent), removes the user account if it exists.  When C(query) (new in 1.9), retrieves the user account details without making any 
changes.'\nlib/ansible/modules/windows/win_user.py:0:0: E305 DOCUMENTATION.options.update_password.description.0: expected a list @ data['options']['update_password']['description'][0]. Got 'C(always) will update passwords if they differ.  C(on_create) will only set the password for newly created users.'\nlib/ansibl
e/modules/windows/win_user.py:0:0: E305 DOCUMENTATION.options.user_cannot_change_password.description.0: expected a list @ data['options']['user_cannot_change_password']['description'][0]. Got 'C(yes) will prevent the user from changing their password.  C(no) will allow the user to change their password.'\nlib/ansib
le/modules/windows/win_webpicmd.py:0:0: E305 DOCUMENTATION.options.name.description.0: expected a list @ data['options']['name']['description'][0]. Got 'Name of the package to be installed'\n```\n<!-- job_id: 58ca60fbbec32a0500f92705 -->\n<!-- job_id: 58ca60fc2e042a06002c9835 -->\n<!-- job_id: /testresults/ansible-t
est-sanity-validate-modules.json -->\n\n[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)\n<!--- boilerplate: shippable_test_result --->"} ==> 422 {'status': '422 Unprocessable Entity', 'content-length': '248', 'x-github-media-type': 'github.v3; format=json', 'x-content-type
-options': 'nosniff', 'content-security-policy': "default-src 'none'", 'access-control-expose-headers': 'ETag, Link, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval', 'x-github-request-id': '8278:DCFD:46462CE:59FD2DB:58CA62FB', 'stri
ct-transport-security': 'max-age=31536000; includeSubdomains; preload', 'x-ratelimit-remaining': '4926', 'server': 'GitHub.com', 'x-ratelimit-limit': '5000', 'date': 'Thu, 16 Mar 2017 10:03:39 GMT', 'x-frame-options': 'deny', 'access-control-allow-origin': '*', 'content-type': 'application/json; charset=utf-8', 'x-x
ss-protection': '1; mode=block', 'x-ratelimit-reset': '1489661692'} {"message":"Validation Failed","errors":[{"resource":"IssueComment","code":"custom","field":"body","message":"body is too long (maximum is 65536 characters)"}],"documentation_url":"https://developer.github.com/v3/issues/comments/#create-a-comment"}
422 {u'documentation_url': u'https://developer.github.com/v3/issues/comments/#create-a-comment', u'message': u'Validation Failed', u'errors': [{u'field': u'body', u'message': u'body is too long (maximum is 65536 characters)', u'code': u'custom', u'resource': u'IssueComment'}]}
2017-03-16 10:03:41,366 ERROR breakpoint!
> /home/ansibot/ansibullbot/lib/decorators/github.py(114)inner()
-> logging.warning('sleeping %s minutes' % (stime/60))
```

--- a/lib/triagers/ansible.py
+++ b/lib/triagers/ansible.py
@@ -899,9 +899,10 @@ class AnsibleTriage(DefaultTriager):
                 logging.debug(e)
                 import epdb; epdb.st()
 
-            #import epdb; epdb.st()
-            if comment not in self.actions['comments']:
-                self.actions['comments'].append(comment)
+            # https://github.com/ansible/ansibullbot/issues/423
+            if len(comment) < 65536:
+                if comment not in self.actions['comments']:
+                    self.actions['comments'].append(comment)
 
         # https://github.com/ansible/ansibullbot/issues/293
         if self.issue.is_pullrequest():

